src/Symfony/Bundle/FrameworkBundle/Tests/Command/XliffLintCommandTest.php

```diff
@@ -64,7 +64,7 @@ public function testLintFilesFromBundleDirectory()
     {
         $tester = $this->createCommandTester($this->getKernelAwareApplicationMock());
         $tester->execute(
-            array('filename' => '@AppBundle/Resources'),
+            array('filename' => array('@AppBundle/Resources')),
```

Comment:
although not ideal, using `class_exists(IntlMessageFormatter::class) ? '...' : array('...')` could fix the deps=high build

Reply:
hmm no actually... we would need to patch 4.1